package.json

```diff
@@ -51,9 +51,6 @@
     "solidity-docgen": "^0.5.3",
     "truffle": "^5.1.33"
   },
-  "resolutions": {
```

Comment:
Why this change?

Reply:
This is already outdated so resolving this issue.